### PR TITLE
NVyrsta

### PR DESCRIPTION
**Pull request by NVyrsta**
[DEMO LINK](https://NVyrsta.github.io/react_people-table-basics/)

### CODE REVIEW

**Review by svitjojo — APPROVED** [review]
Good job 👍

File: src/pages/PeoplePage/PeoplePage.tsx [hunk 1]
```diff
@@ -0,0 +1,59 @@
+import React, { useEffect, useState } from 'react';
+import { useParams } from 'react-router-dom';
+import { Loader } from '../../components/Loader';
+import { Person } from '../../types';
+import { getPeople } from '../../api';
+import { PeopleTable } from '../../components/PeopleTable';
+
+export const PeoplePage: React.FC = () => {
```
Comment (ArturErmolenko) [inline src/pages/PeoplePage/PeoplePage.tsx:8]: memo

File: src/components/PersonLink/PersonLink.tsx [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+            <Link
+              to={`/people/${mother.slug}`}
+              className={classNames({
+                'has-text-danger': mother.sex === 'f',
+              })}
+            >
+              {mother.name}
+            </Link>
+          )
+          : (motherName || '-')}
```
Comment (ArturErmolenko) [inline src/components/PersonLink/PersonLink.tsx:62]: ternary operator should return one value depending on condition, don't make another condition in the ternary operator

File: src/components/PersonLink/PersonLink.tsx [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+      </td>
+
+      <td>
+        {father
+          ? <Link to={`/people/${father.slug}`}>{father.name}</Link>
+          : (fatherName || '-')}
```
Comment (ArturErmolenko) [inline src/components/PersonLink/PersonLink.tsx:68]: same here use variable

File: src/components/PersonLink/PersonLink.tsx [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+import { Person } from '../../types';
+
+type Props = {
+  person: Person;
+  selectedPerson: string;
+  mother: Person | null;
+  father: Person | null;
+};
+
+export const PersonLink: React.FC<Props> = ({
```
Comment (ArturErmolenko) [inline src/components/PersonLink/PersonLink.tsx:13]: memo

**Review by ArturErmolenko — CHANGES_REQUESTED** [review]

**Pull request closed without merging**